### PR TITLE
grafana readme

### PR DESCRIPTION
**Pull request by uwedeportivo**
describes what to do about the csrf bug

### CODE REVIEW

**Comment by uwedeportivo** [comment]
hold off for now, got new information, need to incorporate

**Comment by uwedeportivo** [comment]
ready for review

File: base/grafana/README.md [hunk 1]
```diff
@@ -2,9 +2,33 @@
...
 
 If you are deploying Sourcegraph to a non-default namespace, you'll have to change the namespace specified in
  [grafana.ClusterRoleBinding.yaml](grafana.ClusterRoleBinding.yaml) to the one that you created.
+
+## Docker image
+
+We are running our own image of Grafana which contains a standard Grafana installation packaged together with provisioned dashboards.
+For details see [Grafana Docker Image](https://github.com/sourcegraph/sourcegraph/blob/master/docker-images/grafana/README.md)
+
+## Exposing Grafana directly
```
Comment (slimsag) [inline base/grafana/README.md]: These docs should go into https://docs.sourcegraph.com/admin/monitoring_and_tracing not here given admins will need them. Anything in this dir should be specific to devs

Comment (uwedeportivo) [reply base/grafana/README.md]: i see, ok. i'll move

**Comment by uwedeportivo** [comment]
will move changes to docs

**Pull request closed without merging**